mappings/net/minecraft/world/chunk/ReadOnlyChunk.mapping

```diff
@@ -1,8 +1,9 @@
 CLASS net/minecraft/unmapped/C_riffdstx net/minecraft/world/chunk/ReadOnlyChunk
```

Comment:
(See #26) any suggestions?

Reply:
Given the fact that it is currently named `ReadOnlyChunk` I'd like to hope there is some grounds to this. As said in #26 it can be a little confusing. To me this says that chunks like this can "be locked" so they're ReadOnly, or "unlocked" so they can be written to. Something around this concept perhaps?

Reply:
(Write)LockableChunk?